<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimplexWindow</class>
 <widget class="QDialog" name="SimplexWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>576</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Simplex method</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QScrollBar::vertical{
background: none;
border-style: outset;
border-width: 1px;
border:none;
width:7px;
border-radius: 3px;
}
QScrollBar::horizontal
{
background: none;
border-style: outset;
border-width: 1px;
border:none;
height:7px;
border-radius: 3px;
}
QScrollBar::handle:vertical,QScrollBar::handle:horizontal {
background-color: #808080;
border-style: outset;
border-width: 1px;
border-radius: 3px;
width:7px;
}

QScrollBar::sub-line:vertical,QScrollBar::sub-line:horizontal {
	border: none;
      background: none;

}


/* BTN BOTTOM - SCROLLBAR */
QScrollBar::add-line:vertical,QScrollBar::add-line:horizontal {
		border: none;
      background: none;
}
QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical,QScrollBar::left-arrow:horizontal, QScrollBar::right-arrow:horizontal {
	background: none;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical,QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
	background: none;
}

QHeaderView::section
{

    background-color: #808080;
/*	color:#F5F5DC; */
    padding: 4px;
    font:bold 14px;

}
QHeaderView::section:checked
{
    background-color:FFFFFF;
	
    
}
QTableView
{
    font:/*bold*/ 14px;

}
QTableView::item
{
	background-color: #d9d9d9;
/*	color:#F5F5DC; */
}
QTableView::item::selected
{
	background-color: #464646;
    
}

QTableView QTableCornerButton::section
{
    background-color: #808080;
    
}

</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000000</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#F8F8FF</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>7</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #808080;
color:#F5F5DC;
 border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
padding: 5px</string>
            </property>
            <property name="text">
             <string>Number of variables:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #FFFFFF;
color:#8c8c8c;
 border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
padding: 5px</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="vars_add">
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #808080;
color:#F5F5DC;
 border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
padding: 5px</string>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="vars_remove">
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #808080;
color:#F5F5DC;
 border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
padding: 5px</string>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #808080;
color:#F5F5DC;
 border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
padding: 5px</string>
            </property>
            <property name="text">
             <string>Number of constraints:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #FFFFFF;
color:#8c8c8c;
 border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
padding: 5px</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="constraints_add">
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #808080;
color:#F5F5DC;
 border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
padding: 5px</string>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="constraints_remove">
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #808080;
color:#F5F5DC;
 border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
padding: 5px</string>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>167772</width>
          <height>150000</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>960</width>
           <height>329</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2"/>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #808080;
color:#F5F5DC;
 border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
padding: 5px</string>
        </property>
        <property name="text">
         <string>Generate random input</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnrun">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #808080;
color:#F5F5DC;
 border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    font: bold 14px;
padding: 5px</string>
        </property>
        <property name="text">
         <string>Run algorithm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitbtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>9</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
